src/providers/ipa/ipa_s2n_exop.c

```diff
@@ -1454,10 +1482,11 @@ struct tevent_req *ipa_s2n_get_acct_info_send(TALLOC_CTX *mem_ctx,
         goto fail;
     }
 
-    DEBUG(SSSDBG_TRACE_FUNC, "Sending request_type: [%s] for trust user [%s] "
-                            "to IPA server\n",
-                            ipa_s2n_reqtype2str(state->request_type),
-                            req_input->inp.name);
+    input = ipa_s2n_reqinp2str(state, req_input);
+    DEBUG(SSSDBG_TRACE_FUNC, "Sending request_type: [%s] for trust user "
+          "[%s] to IPA server\n", ipa_s2n_reqtype2str(state->request_type),
+          input);
+    talloc_zfree(input);
```

Comment:
ipa_s2n_reqinp2str() should only be called if input is needed. Se I would recommend to either wrap the debug block in a 'if (DEBUG_IS_SET(SSSDBG_TRACE_FUNC))' or move ipa_s2n_reqinp2str() into the DEBUG() macro and drop the individual talloc_free() here.